creator-node/package.json

```diff
@@ -13,7 +13,8 @@
     "test:coverage": "nyc --reporter=lcov --reporter=text npm run test",
     "test:coverage:ci": "nyc --reporter=lcov --reporter=text npm run test:ci && nyc report --reporter=text-lcov | coveralls",
     "lint:fix": "eslint --fix --ext=js,ts src",
-    "lint": "eslint --ext=js,ts src"
+    "lint": "eslint --ext=js,ts src",
+    "build": "./node_modules/.bin/tsc --project ./"
```

Comment:
why do this?

Reply:
unrelated to this work but useful to test that the build works before starting the whole `A up` flow to ensure no type errors
is there a better way to do this?

Reply:
no just wanted to figure out where this change fits in. fine with keeping it in and seems reasonable

Reply:
kk